src/views/DashboardView/StudentDashboard/index.vue

```diff
@@ -1,6 +1,13 @@
 <template>
   <div class="student-dashboard">
     <dashboard-banner />
+    <div
```

Comment:
Might have been good if I had made this into a component since it's also being used in the volunteer dashboard